srcpkgs/catatonit/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'catatonit'
+pkgname=catatonit
+version=0.1.5
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool"
+short_desc="Container init that is so simple it's effectively brain-dead"
+maintainer="Justin Bronder <jsbronder@cold-front.org>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/openSUSE/catatonit"
+distfiles="https://github.com/openSUSE/catatonit/releases/download/v${version}/catatonit.tar.xz"
+checksum=4c45e2230235751731da3daabe130abce70b3ef37d0b1936cbc90a46b475cf08
+
+pre_configure() {
+	./autogen.sh
+}
+
+do_install() {
+	vmkdir usr/libexec/catatonit
+    vinstall catatonit 0755 usr/libexec/catatonit
+
+	vmkdir usr/libexec/podman
+	ln -sf ../catatonit/catatonit "${DESTDIR}/usr/libexec/podman"
```

Comment:
Explain why this is necessary.

Reply:
Added a comment in the template.  For convenience:
```
    # Following suit with Fedora/Ubuntu, podman-run will default to looking
    # for catatonit here when passed the "--init" argument
```